validator_client/src/attestation_service.rs

```diff
@@ -428,6 +429,7 @@ impl<T: SlotClock + 'static, E: EthSpec> AttestationService<T, E> {
                 log,
                 "Successfully published attestations";
                 "count" => attestations.len(),
+                "validator_indices" => format!("{:?}", validator_indices),
```

Comment:
Inside the `slog` logging macros use of the `Debug` trait can be abbreviated using `?`

```suggestion
                "validator_indices" => ?validator_indices,
```